tests/test_datasets/test_internal_datasets.py

```diff
@@ -126,13 +126,97 @@ def test_not_present_part():
             pd.to_datetime("2022-01-01 00:00:00"),
             ("train", "test"),
         ),
+        (
+            "m3_monthly",
+            (144, 2856),
+            pd.to_datetime("2010-01-31 00:00:00"),
+            pd.to_datetime("2021-12-31 00:00:00"),
+            ("train", "test"),
+        ),
+        (
+            "m3_quarterly",
+            (72, 1512),
+            pd.to_datetime("2004-03-31 00:00:00"),
+            pd.to_datetime("2021-12-31 00:00:00"),
+            ("train", "test"),
+        ),
+        (
+            "m3_other",
+            (104, 348),
+            pd.to_datetime("1996-03-31 00:00:00"),
+            pd.to_datetime("2021-12-31 00:00:00"),
+            ("train", "test"),
+        ),
+        (
+            "m3_yearly",
+            (47, 1290),
+            pd.to_datetime("1975-12-31 00:00:00"),
+            pd.to_datetime("2021-12-31 00:00:00"),
+            ("train", "test"),
+        ),
     ],
 )
 def test_dataset_statistics(dataset_name, expected_shape, expected_min_date, expected_max_date, dataset_parts):
+    import ssl
```

Comment:
Why do we need this?

Reply:
exception is raised while doing pd.read_csv from url. ssl helps

Reply:
I think we should investigate this and try to avoid this hack.